app/common/validation/PropertyValidation.scala

```diff
@@ -48,4 +48,20 @@ object PropertyValidation {
         broughtForwardLosses, annualExemptAmount, disposalDate)))
     }
   }
+
+  def validatePropertyTaxOwed(propertyCalculateTaxOwedModel: PropertyCalculateTaxOwedModel): Either[String, PropertyCalculateTaxOwedModel] = {
+    val propertyChargeableGainModel = validatePropertyChargeableGain(propertyCalculateTaxOwedModel.propertyChargeableGainModel)
+    val previousTaxableGain = validateOptionDouble(propertyCalculateTaxOwedModel.previousTaxableGain, residentPropertyKeys.previousTaxableGain)
+    val previousIncome = validateDouble(propertyCalculateTaxOwedModel.previousIncome, residentPropertyKeys.previousIncome)
+    val disposalDate = validateDisposalDate(propertyCalculateTaxOwedModel.propertyChargeableGainModel.disposalDate)
+    val personalAllowance = disposalDate match {
```

Comment:
with the removal of disposal date you could update this to personalAllowance = propertyChargeableGainModel.disposalDate